packages/docs-site/src/content/docs/taiko-alethia-protocol/protocol-design/based-preconfirmation.mdx

```diff
@@ -5,21 +5,21 @@ description: Core concept page for "Based Preconfirmations".
 
 import { Aside } from '@astrojs/starlight/components';
 
-Taiko Alethia is implementing based preconfirmations to enhance transaction finality, reducing the current 12-second block time to potentially sub-second finality. This significantly improves UX, particularly for applications requiring fast transaction execution. Based preconfirmations allow transactions to receive early inclusion guarantees from preconfirmers, providing users with immediate execution assurance.
+Taiko Alethia is implementing based preconfirmations to enhance transaction finality, reducing the current 12-second block time to potentially sub-second finality. This implementation is achieved through cryptographic promises from preconfers, secured by stake to encourage promise fulfillment. This significantly improves UX, particularly for applications requiring fast transaction execution. Based preconfirmations allow users to receive a promise of early inclusion from preconfers, providing users with a certain degree of assurance of immediate execution.
```

Comment:
should we replace "promise" with "commitment"?

Reply:
I think this is better than "indication", would prefer this if possible @smartprogrammer93

Reply:
"Commitment" will also have legal implications